### PR TITLE
doc: zebra start command config

### PR DESCRIPTION
**Pull request by danielvladco**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,7 @@ and for Zebra:
 ```toml
 kind = "zebra"
 path = "path/to/zebra/repo"
-start_command = "cargo +stable r -- --verbose start"
+start_command = "target/release/zebrad --verbose"
 # cargo +stable r   run Zebra using stable Rust
 # --                all args after this will get passed to Zebra
```
Comment (Rqnsom) [inline README.md:84]: These comments can be removed

File: README.md [hunk 2]
```diff
@@ -79,7 +79,7 @@ and for Zebra:
 ```toml
 kind = "zebra"
 path = "path/to/zebra/repo"
-start_command = "cargo +stable r -- --verbose start"
```
Comment (Rqnsom) [inline README.md:82]: Do we need the `start` command at the end? Probably not if it works without it

line 86:
```
# start starts the node
```

File: README.md (hunk 1 above)
Comment (danielvladco) [reply README.md]: removed the comments

**Comment by Rqnsom** [comment]
Not sure how to comment on the commit message directly,
But I would say this is not a `fix` but a `doc` update 🤔
It's a fix in a doc update

File: README.md (hunk 2 above)
Comment (Rqnsom) [reply README.md:82]: Can you test with the `start` command here?

Their help text is not very correct here:

```
➜  src git:(main) zebrad fsdfds                                                                                                                                                                                                                              git:(main↓2↑3|⚑2)
error: unrecognized command `fsdfds`

zebrad 1.0.0-rc.0
Zcash Foundation <zebra@zfnd.org>

FLAGS:
    -c, --config CONFIG       path to configuration file
    -h, --help                print help message
    -v, --verbose             be verbose

➜  src git:(main) zebrad help                                                                                                                                                                                                                                git:(main↓2↑3|⚑2)
zebrad 1.0.0-rc.0
Zcash Foundation <zebra@zfnd.org>

USAGE:
    zebrad <SUBCOMMAND>

SUBCOMMANDS:
    copy-state copy cached chain state (debug only)
    download   pre-download required parameter files
    generate   generate a skeleton configuration
    help       get usage information
    start      start the application
    tip-height get the block height of Zebra's persisted chain state
    version    display version information

➜  src git:(main) zebrad --help                                                                                                                                                                                                                              git:(main↓2↑3|⚑2)
2022-10-28T06:19:56.113995Z  INFO zebrad::components::tracing::component: started tracing component filter="info" TRACING_STATIC_MAX_LEVEL=LevelFilter::INFO LOG_STATIC_MAX_LEVEL=Info
2022-10-28T06:19:56.114017Z  INFO zebrad::application: Diagnostic Metadata:
[...]
```

`zebrad help` works, but it isn't documented,
`zebrad --help` is documented, but it doesn't work 😀

**Comment by Rqnsom** [comment]
> Not sure how to comment on the commit message directly,
But I would say this is not a fix but a doc update 🤔
It's a fix in a document

So, can we change the commit tag to `doc`? `fix` implies a bug, and we don't want to show a bug in the git tree falsely 🥲

Otherwise, it's LGTM, but but I don't have any approval access (or merge access, or resolve comment access)

**Pull request closed without merging**